__tests__/Util-test.ts

```diff
@@ -29,12 +30,16 @@ describe('getUserData util', () => {
     username: 'anish-pawaskar',
   };
 
-  test.skip('when url passed !== redirect url return null', async () => {
-    const res = await getUserData('https://www.example.net/');
-    expect(res).toEqual(null);
+  test('when token passed is invalid, axios call returns with 401', async () => {
+    mockedAxios.get.mockRejectedValue('401: Unauthorized');
+    try {
+      await getUserData(invalidToken);
```

Comment:
Is the function accepting token or userId?

Reply:
token